EnyimMemcached/NHibernate.Caches.EnyimMemcached/NHibernate.Caches.EnyimMemcached.csproj

```diff
@@ -1,37 +1,37 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Full changed file again for no apparent reason.
I speculate the file is saved in GitHub with unexpected line endings, but the line ending normalization occurring on checkout hides this, until we do any change in it.
I think "eating the bullet" once should solve this.
Otherwise we would have to edit this file (and others having the same trouble) only through the GitHub web site to dodge the issue, which is a bit unpractical.